docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -77,12 +77,18 @@ Before you install YugabyteDB on a Kubernetes cluster, perform the following:
 
 ### Service account
 
-The secret of a service account can be used to generate a `kubeconfig` file. This account should not be deleted once it is in use by YugabyteDB Anywhere. *namespace* in the service account creation command can be replaced with the desired namespace in which to install YugabyteDB.
+The secret of a service account can be used to generate a `kubeconfig` file. This account should not be deleted once it is in use by YugabyteDB Anywhere. 
+
+Set the `YBA_NAMESPACE` environment variable to the namespace where you have installed YugabyteDB Anywhere. This variable will used by the rest of the commands from this document.
+
+```sh
+export YBA_NAMESPACE="yb-platform"
```

Comment:
though repetitive, it would be better to include this export in each shell command so that folks can copy-paste directly.

Reply:
Yep, but the only problem with that is the users have to modify the environment variable everytime for each command if they are using a different value than what we have recommended in this document.

Reply:
So, I checked what happens when we don't set the value and run the commands. None of them succeeds but give weird errors, which will make it hard to debug. 
I'm adding default value to all the variable expansions. 
```
${YBA_NAMESPACE:-yb-platform}
```
That way if they provide anything, it will default to yb-platform. 
What do you think about this idea?

Reply:
Makes sense.